app/public/fullcalendar-5.5.1/lib/locales/af.js

```diff
@@ -0,0 +1,27 @@
+FullCalendar.globalLocales.push(function () {
```

Comment:
Are all of these locales necessary as well? I assume so in case clients can only read or prefer other languages right?